src/components/custom/_signup-new.js

```diff
@@ -193,13 +192,19 @@ const SignupNew = ({
 }) => {
     const [checkBoxState, setCheckBoxState] = useState(false)
 
+    // Correct
```

Comment:
LGTM, but shall we remove this line of comments?

Reply:
Removed and thanks